ADOPTERS.md

```diff
@@ -0,0 +1,4 @@
+| Organization                                                             | Contact                                                                                                                                                                                                      | Description of Use                                                                                                                                                                                                                                                                                                                                   |
+| ------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Gitpod](https://www.gitpod.io)                                          | [www.gitpod.io/chat](https://www.gitpod.io/chat)                                                                                                                                                             | Gitpod builds Gitpod with Gitpod on Gitpod.                                                                                                                                                                                                                                                                                                          |
+
```

Comment:
[The Backstage one](https://github.com/backstage/backstage/blob/master/ADOPTERS.md) also has a column for an optional contact? Should/could we add that also? That way if companies want to get in touch with others about how they're using Gitpod that they can do it? Just to further promote the cross-company communication?